### DISCUSSION TITLE
Database file

### DISCUSSION THREAD

**Discussion by anjali7786** [vigneshshettyin/Bulk-Mailer#36] [Q&A] [answered]
Hi, I followed the steps to do the setup of the application on my local machine. I am using Pycharm but I am not able to see the bulkmailer.db file. Please tell me how I would be able to get access to it.

**Comment by vigneshshettyin** [comment, marked as answer]
First check import.json `"databaseUri": "sqlite:///bulkmailer.db",`
Then if bulkmailer.db file is not found then type these commands in python shell
1.) `from app import db`
2.) `db.create_all()`

**Reply by kanishka8276** [reply]
@vigneshshettyin I can find the bulkmailer.db file in the main folder but on registering or logging in it shows sqlite3.Operationalerror 
No such table:user

**Reply by vigneshshettyin** [reply]
Once you have cloned the recent repo and virtual environment is created successfully. Then check this video for further setup of app (https://www.loom.com/share/39e7161c97984ae48dadc30838edbe6c) @kanishka8276
